front/src/components/DoranAppBar.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <v-app-bar flat>
+      <a href="#">
+        <i class="mdi mdi-chevron-left"></i>
+      </a>
+      <div class="location-title">
+        <i class="mdi mdi-map-marker pr-1"></i>
+        <v-toolbar-title id="current-location"
+          >서울특별시 송파구 가락본동</v-toolbar-title
+        >
```

Comment:
닫는 태그는 한 줄에 써주는 것이 어떨까요?
```suggestion
          >서울특별시 송파구 가락본동
          </v-toolbar-title>
```

Reply:
prettier을 적용했더니 이렇게 정리가 됩니다!
`prettierrc.js` 파일에서 `jsxBracketSameLine: false,`로 설정이 되어 있어서 그런 것 같은데 이부분은 회의 때 다시 논의해볼까요?